generators/gae/templates/app.yaml.ejs

```diff
@@ -0,0 +1,20 @@
+runtime: java11
+instance_class: <%= gaeInstanceClass %>
+service: <%=gaeServiceName%>
+<%_ if (gaeScalingType === 'manual') { _%>
+manual_scaling:
+    instances: <%= gaeInstances %>
+<%_ } else if (gaeScalingType === 'basic') { _%>
+basic_scaling:
+    <%_ if (gaeMaxInstances > 0) { _%>
+    max_instances: <%= gaeMaxInstances %>
+    <%_ } _%>
+<%_ } else if (gaeScalingType === 'automatic') { _%>
+automatic_scaling:
```

Comment:
I have an error when answering the default values in the prompt of `min_instance: 0` and `max_instance: 0`, in this cas the `automatic_scaling:` block should disappear entirely. https://cloud.google.com/appengine/docs/flexible/java/reference/app-yaml#automatic_scaling

Reply:
I've made changes to this. I think one should be allowed to choose max_instances: 0 and/or min instances: 0. These are allowed settings in GCP and I don't think we need the block to disappear. If max_instances: 0 is chosen the setting will be disabled anyways. I've made an error in my previous commit such that when 0 is chosen the max/min_instances blocks doesn't appear. 

Let me know your thoughts; I also see that the reference doc you pointed seems to the flexible environment not the standard one which is what we support; the correct doc is; https://cloud.google.com/appengine/docs/standard/java11/config/appref#automatic_scaling
The way standard handles max/min_instances is a bit different. 

Supporting flexible is a different discussion which I'll bring up in perhaps another PR. 😄

Reply:
Yes you are correct on both points.